docs/examples/blind-auction.rst

```diff
@@ -184,7 +184,7 @@ invalid bids.
 
 ::
 
-    pragma solidity >0.4.23 <0.7.0;
+    pragma solidity >0.4.99 <0.7.0;
```

Comment:
Why not `>=0.4.24` or `>=0.5.0`?

Reply:
example has features from 0.5.0, so going with >=0.5.0